src/material-experimental/mdc-list/list-option.html

```diff
@@ -15,9 +15,9 @@
       no accidental change event leaks out of the list option or selection list when
       the checkbox is directly clicked.
     -->
-    <input type="checkbox" tabindex="-1" class="mdc-checkbox__native-control"
-           [checked]="selected" [disabled]="disabled" [attr.aria-describedby]="_optionTextId"
-           (change)="$event.stopPropagation()" />
+    <input type="checkbox" class="mdc-checkbox__native-control"
+           [checked]="selected" [disabled]="disabled"
+           (change)="$event.stopPropagation()"/>
```

Comment:
do we still need this change listener if it's `display:none` now?

Reply:
You're right, we probably don't need it. Removing.